android-ktx/build.gradle

```
@@ -40,12 +40,13 @@ configurations.all {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    api 'com.mirego.trikot:viewmodels:1.0.6'
+    api 'com.mirego.trikot:viewmodels:1.0.11-SNAPSHOT'
```

Je pense qu'il faut que tu le remette au latest ici?

```suggestion
    api 'com.mirego.trikot:viewmodels:1.0.10
```

---

The latest for android-ktx is `1.0.6`. 1.0.10 is the viewModel version

---

Ohh nevermind it's precisely the dependency to viewModel. So yeah I think we have to set 'com.mirego.trikot:viewmodels:1.0.11' but we have to do it after mirego-builds has done his `mirego.release pre tag commit` commit

---

doing so renders me unable to build locally and make sure everything works. how do i circumvent the problem?

---

@npresseault could confirm, but I think our usual process is once we are sure every works locally, we simply "unsnapshot" our versions and merge our work. The next release build will then have the proper versions to work with.